testsuite/MDAnalysisTests/datafiles.py

```diff
@@ -187,7 +187,9 @@
     "GMX_TOP_BAD", # file with an #include that doesn't exist
     "ITP_no_endif", # file missing an #endif
     "PDB_CRYOEM_BOX", #Issue 2599
-    "PDB_CHECK_RIGHTHAND_PA" # for testing right handedness of principal_axes
+    "PDB_CHECK_RIGHTHAND_PA", # for testing right handedness of principal_axes
+    "PDB_NOCRYST", # File with meaningless CRYST1 record (Issue #2679, PR #2685)
```

Comment:
A thought here, probably not worth too much attention. In order to reduce the amount of data we have to store, would it make sense to use `PDB_CRYOEM_BOX` instead of `PDB_NOCRYST`? I realise that 2BBM was what was reported in the original issue and that it's NMR whilst the other Cryo-EM, but we are looking for the same CRYST1 record right? Alternatively a tiny NMR structure could be used like 6QYR.

Reply:
Yes, @orbeckst mentioned something on those lines before (and I forgot):

> Bundle the file as bzip2-compressed file or even better, just a minimal example.

I'll switch the `PDB_NOCRYST` with `PDB_CRYOEM_BOX` and change the MMTF file to something smaller (`6QYR`).